website/templates/project/project_base.mako

```diff
@@ -14,6 +14,11 @@
 
 <%include file="project_header.mako"/>
 
+% if status and node:
+    ## WARNING: Need new heading alert messages? They must be added at both here and base.mako
```

Comment:
@sloria The situation I consider is when we immigrant the website and there is a heading on whole website saying "our service will not be available at some time". I noticed that  message was on the top of the if statement. In that case, it will get the image similar like this:
![screenshot 2015-09-23 09 31 57](https://cloud.githubusercontent.com/assets/5420789/10046301/030d187c-61d6-11e5-87df-30af8d304dd5.png)